### ISSUE TITLE
A way to search for folders or for files with specified attributes

### ISSUE THREAD

**Issue by euglv** [szwacz/fs-jetpack#20]
Can I search for folders, like I do for files https://github.com/szwacz/fs-jetpack#find ?
It would be great idea to extend `searchOptions` in `find` method to allow searching for folders and filtering files by file attributes.

**Comment by szwacz** [comment]
Makes sense. The best I can think of from the top of my head is to provide switches:

```js
jetpack.find('foo', { matching: '*bar', files: false, dirs: true });
```
`files` by default will be `true` and `dirs` by default `false`.

Does this get the job done for you?

**Comment by euglv** [comment]
This would be great

**Comment by szwacz** [comment]
Just released version 0.8.0 has support for files and directories finding.
